ros/api/v1/hosts.py

```diff
@@ -30,25 +30,32 @@ def get(self):
         ident = identity(request)['identity']
         account_query = db.session.query(RhAccount.id).filter(
             RhAccount.account == ident['account_number']).subquery()
-        system_count = db.session.query(System.id)\
-            .filter(System.account_id.in_(account_query)).count()
-        systems_with_suggestions = db.session.query(System.id)\
-            .filter(System.account_id.in_(account_query))\
-            .filter(System.number_of_recommendations > 0).count()
-        systems_waiting_for_data = db.session.query(System.id)\
-            .filter(System.account_id.in_(account_query))\
-            .filter(System.state == 'Waiting for data').count()
+        system_query = db.session.query(System.id).filter(System.account_id.in_(account_query))
+        last_reported = (
```

Comment:
@apuntamb  - Could you please add new issue for checking PerformanceProfile records?
Original issue only talks about inconsistent response formats.